control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{ApiClientError, CreateVolumeTopology, CsiControllerConfig, IoEngineApiClient};
 
-use csi_driver::context::{CreateParams, PublishParams};
+use csi_driver::context::{CreateParams, PublishParams, IS_CLONE};
```

Comment:
Do we need the bool IS_CLONE? Can this not be deciphered from the CreateParams?

Reply:
No, CreateParams are the ones that get passed from StorageClass. There is no storageclass parameter that tells us the volume would be a clone. So have added this to publish context in the publish flow.

Reply:
Instead of IS_CLONE, should we consider passing the VolumeSource here? 
- If the VolumeSource is empty, it's a regular volume.
- If the VolumeSource is a snapshot, it's a volume restore.
- If the VolumeSource is another volume, it's a volume clone.

Reply:
Yes, we can do that as well.

Reply:
Have removed it as discussed.